src/sentry/static/sentry/app/views/performance/realUserMonitoring/types.tsx

```diff
@@ -11,6 +11,7 @@ export type Vital = {
   description: string;
   failureThreshold: number;
   type: ColumnType;
+  display: boolean;
```

Comment:
This is just here temporarily right?

Reply:
yea. don't want to load them as histograms.